core/src/main/resources/templates/email_already_registered.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Email Already Registered</title>
+</head>
+<body>
+<script>
+alert("This email address is already a registered user")
```

Comment:
This should not be a different page. See how you can show the error message without changing the page

Reply:
this has been implemented